src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -286,46 +317,72 @@ export class BrowsingContextProcessor {
   async process_script_addPreloadScript(
     params: Script.AddPreloadScriptParameters
   ): Promise<Script.AddPreloadScriptResult> {
-    const contexts: BrowsingContextImpl[] = [];
-    const scripts: Script.PreloadScript[] = [];
+    const cdpTargets = new Set<CdpTarget>(
+      params.context === undefined || params.context === null
+        ? this.#browsingContextStorage
+            .getTopLevelContexts()
+            .map((context) => context.cdpTarget)
+        : [this.#browsingContextStorage.getContext(params.context).cdpTarget]
```

Comment:
Because of the OOPIF as well:
```suggestion
        : [this.#browsingContextStorage.getContext(params.context).cdpTarget, ...this.#browsingContextStorage.getContext(params.context).flattenAllChildren]
```